makeflow/src/dag_file.c

```diff
@@ -84,4 +87,32 @@ int dag_file_in_trans( const struct dag_file *f )
 		return 0;
 }
 
+int dag_file_mount_clean(struct dag *d) {
```

Comment:
Makeflow has lots of places doing memory allocation and free, and I do not think I know all of them. 
When I implement my branch, I try to guarantee all the codes I wrote should have no memory leakage.